library/core/src/main/java/com/google/android/exoplayer2/video/MediaCodecVideoRenderer.java

```diff
@@ -969,6 +983,10 @@ protected MediaFormat getMediaFormat(
     // Set codec configuration values.
     if (Util.SDK_INT >= 23) {
       mediaFormat.setInteger(MediaFormat.KEY_PRIORITY, 0 /* realtime priority */);
+      if (format.frameRate != Format.NO_VALUE) {
+        float codecOperatingRate = getCodecOperatingRate();
```

Comment:
Can we explicitly pass operating rate down through `configureCodec` and into `getMediaFormat`, to avoid the need for this getter?